packages/shared/src/components/tooltips/SimpleTooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useMemo } from 'react';
+import dynamicParent from '../../lib/dynamicParent';
+import { getShouldLoadTooltip, BaseTooltipProps } from './BaseTooltip';
+
+const BaseTooltipLoader = () =>
+  import(/* webpackChunkName: "lazyTooltip" */ './BaseTooltip');
+
+export type SimpleTooltipProps = Pick<
+  BaseTooltipProps,
+  'content' | 'children' | 'placement'
+>;
+
+const TippyTooltip = dynamicParent<SimpleTooltipProps>(
+  () => BaseTooltipLoader().then((mod) => mod.BaseTooltip),
+  React.Fragment,
+);
+
+export function SimpleTooltip({
+  children,
+  content,
+  ...props
+}: SimpleTooltipProps): ReactElement {
+  const component = useMemo(
+    () =>
+      React.cloneElement(children, {
+        ...children.props,
+        'aria-label': typeof content === 'string' && content,
+      }),
+    [children],
+  );
+
+  return (
+    <TippyTooltip
+      shouldLoad={getShouldLoadTooltip()}
```

Comment:
I think that instead of `shouldLoad`, we can add an if statement at the beginning and if should load is false we can return an empty element.
Something like:
```
if (!shouldLoad) {
  return <></>;
}
```
What do you think?

Reply:
That's a good idea. Though instead of empty `Fragment` I think we should return the children which need to be rendered regardless of tooltip and has the `aria-label` property.

Reply:
This is a good idea. With that, I tried rendering the `children` only if `shouldLoad` is `false` (since it is necessary to be displayed), but tests started failing again even though it still has the `aria-label`. I also implemented a dynamic rendering since it is optional but it seems to crash again on tests.